quantum/rgblight.c

```diff
@@ -797,6 +805,16 @@ void rgblight_task(void) {
     // static light mode, do nothing here
     if ( 1 == 0 ) { //dummy
     }
+#ifdef RGBLIGHT_IDLE_ENABLE
+    // exit early if we are timedout
+    else if (rgblight_idle_timedout) {
+        return;
+    } else if (timer_elapsed32(rgblight_idle_timer) >= (RGBLIGHT_IDLE_TIMEOUT * 60000)) {
```

Comment:
pull magic number into a define please. 60 seconds I assume?

Reply:
Yes, this is to convert the input (in minutes) to the milliseconds of the timer. Fixed here:
https://github.com/qmk/qmk_firmware/pull/6450/files#diff-22424b86d2ea37fb185ca51cc3cb9bb1R812

Reply:
Yes, this is to convert the input (in minutes) to the timer's milliseconds. Fixed in the latest commit:
https://github.com/qmk/qmk_firmware/pull/6450/files#diff-22424b86d2ea37fb185ca51cc3cb9bb1R812

Reply:
Yes, this is to convert the user's input (in minutes) to the timer's milliseconds. Fixed in latest commit.